packages/teams-js/src/public/runtime.ts

```diff
@@ -384,6 +387,7 @@ export function generateBackCompatRuntimeConfig(highestSupportedVersion: string)
 
   const backCompatRuntimeConfig: Runtime = {
     apiVersion: 2,
+    hostVersionsInfo: { adaptiveCardSchemaVersion: minAdaptiveCardVersion },
```

Comment:
this should also be updated to use the same new const (or just pull the value off of teamsRuntimeConfig)

Reply:
Updated in the latest commit to use new const variable.